docs/dapps/integration/contract-integration.md

```diff
@@ -140,6 +140,10 @@ We recommend directing them to this page.
 
 :::
 
+## Combining Data Feeds
```

Comment:
Burak doesn't use the capitalized titles on the docs (apart from service names and coined terms) so this should be just "Combining data feeds"